server/lib/nl/detection/llm_detector.py

```diff
@@ -19,16 +19,67 @@
 from flask import current_app
 
 from server.lib.nl.common import counters
+from server.lib.nl.common import utterance
 from server.lib.nl.detection import palm_api
+from server.lib.nl.detection import types
 from server.lib.nl.detection.place import get_place_from_dcids
 from server.lib.nl.detection.place import infer_place_dcids
 from server.lib.nl.detection.types import Detection
 from server.lib.nl.detection.types import PlaceDetection
 from server.lib.nl.detection.types import SVDetection
 
+# TODO: Add support for COMPARISON_FILTER and RANKING_FILTER
```

Comment:
nit: these type mappings cannot go in to `types.py`?

Reply:
Since these are very LLM prompt specific, thought they can be here for now?